### DISCUSSION TITLE
Create Policyd with Milter, Is there a way to create it?

### DISCUSSION THREAD

**Discussion by jniltinho** [d--j/go-milter#24] [Q&A]
Hi @d--j ,

I need to create a policyd, to add SPF and DNSBL in addition to making some rules that we have within the Mysql database, do you recommend which way to go?

I am using testing these projects,
But I'm having a lot of time out in Postfix, I'm using check_policy_service.

smtpd_recipient_restrictions =
			reject_unauth_destination
			check_policy_service inet:127.0.0.1:10005

https://github.com/jniltinho/postfix-policy-server

**Comment by d--j** [comment]
Sure you can easily have multiple servers in one go process. Have a look at e.g. https://github.com/d--j/srs-milter/blob/main/cmd/srs-milter/main.go – this creates a milter and a socketmap server in one process. You might need to spawn the servers in go routines and wait on them
